commcare_connect/opportunity/management/commands/populate_completed_work_saved_fields.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management import BaseCommand
+
+from commcare_connect.opportunity.models import OpportunityAccess
+from commcare_connect.opportunity.utils.completed_work import _update_status_set_saved_fields_and_get_payment_accrued
+
+
+class Command(BaseCommand):
+    help = (
+        "Populates the saved_* fields on CompletedWork objects "
+        "(and also the payment_accrued field on OpportunityAccess)"
+    )
+
+    def handle(self, *args, **options):
+        for opportunity_access in OpportunityAccess.objects.all():
```

Comment:
The current task applies a bunch of filters on the opportunities it checks, so we will need this management command to backfill the existing data.